test/integration/improve_this_page_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class ImproveThisPageTest < ActionDispatch::IntegrationTest
```

Comment:
Any plans to test when javascript is enabled? Even if only that clicking the feedback link makes the relevant form / message visible?